homeassistant/components/senseme/const.py

```diff
@@ -20,4 +20,10 @@
 SENSEME_DIRECTION_FORWARD = "FWD"
 SENSEME_DIRECTION_REVERSE = "REV"
 
-PLATFORMS = [Platform.BINARY_SENSOR, Platform.FAN, Platform.SWITCH]
+PLATFORMS = [
+    Platform.BINARY_SENSOR,
+    Platform.FAN,
+    Platform.LIGHT,
```

Comment:
Is this a fix?

Reply:
I guess it is. I apparently did a poor job of resolving conflicts in the previous PR.